swift/ql/lib/codeql/swift/generated/expr/ObjectLiteralExpr.qll

```diff
@@ -1,10 +1,51 @@
 // generated by codegen/codegen.py
 private import codeql.swift.generated.Synth
 private import codeql.swift.generated.Raw
+import codeql.swift.elements.expr.Argument
 import codeql.swift.elements.expr.LiteralExpr
 
 module Generated {
+  /**
+   * `#fileLiteral`, `#imageLiteral` and `#colorLiteral` expressions, which are used in playgrounds.
+   */
   class ObjectLiteralExpr extends Synth::TObjectLiteralExpr, LiteralExpr {
     override string getAPrimaryQlClass() { result = "ObjectLiteralExpr" }
+
+    /**
+     * Gets the
```

Comment:
Wat? 😂

Reply:
hmm :laughing: will look into it tomorrow. Also the trailing `.` all alone at the end is weird

Reply:
so it turns out I used `doc` when I wanted to use `desc` :facepalm: I've opened [this PR](https://github.com/github/codeql/pull/11193) to avoid this problem in the future.